### DISCUSSION TITLE
Activate account default email

### DISCUSSION THREAD

**Discussion by laurabennett** [craftcms/cms#12448] [Ideas] [answered]
When setting up users you can have the system send an email with link to create a password. Great!
However, the email doesn't tell them the username they will need to login. Bummer.
Wondering if this could be added to the activation email by default?
Thanks!

**Comment by brandonkelly** [comment, marked as answer]
The current activation message doesn’t really have a place where we could naturally inject the username:

https://github.com/craftcms/cms/blob/af40a2d6e0557f6a600a22f29f1660639cd11b81/src/translations/en/app.php#L1792

However when you click on the activation link, Craft will set a `username` cookie on the browser set to your user account’s username, so the next time you view the login form, the username input will already be filled in.

And if you really want to show the username in the email, you can customize it from Utilities → System Messages.
